### PR TITLE
zh-trans: add docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md

### PR DESCRIPTION
**Pull request by SataQiu**
Fixes: k8smeetup/website-tasks#682

### CODE REVIEW

**Comment by SataQiu** [comment]
/hold
Wait #13170 merged.

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+weight: 40
+---
+-->
+{{% capture overview %}}
+<!--
+`kubeadm upgrade` is a user-friendly command that wraps complex upgrading logic behind one command, with support
+for both planning an upgrade and actually performing it. `kubeadm upgrade` can also be used for downgrading
+cluster if necessary.
+-->
+`kubeadm upgrade` 是一个对用户友好的命令，它将复杂的升级逻辑包装在一个命令后面，支持规划升级和实际执行。
```
Comment (chenrui333) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md]:
```suggestion
`kubeadm upgrade` 是一个对用户友好的命令，它将复杂的升级逻辑包装在一个命令后面，支持升级的规划和实际执行。
```

**Comment by SataQiu** [comment]
@chenrui333  已经修改，多谢！

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+-->
+`kubeadm upgrade` 是一个对用户友好的命令，它将复杂的升级逻辑包装在一个命令后面，支持升级的规划和实际执行。
+如有必要，`kubeadm upgrade` 也可用于降级集群。
+{{% /capture %}}
+
+{{% capture body %}}
+<!--
+## kubeadm upgrade guidance
+-->
+## kubeadm 升级向导
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md]: guidance 似乎译为指南更合适

Comment (SataQiu) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md]: 感谢您的指正，已经修改！

**Pull request merged**